src/main/java/slimeknights/tconstruct/tools/client/particles/AxeAttackParticle.java

```diff
@@ -0,0 +1,33 @@
+package slimeknights.tconstruct.tools.client.particles;
+
+import net.minecraft.client.particle.IAnimatedSprite;
+import net.minecraft.client.particle.IParticleFactory;
+import net.minecraft.client.particle.Particle;
+import net.minecraft.particles.BasicParticleType;
+import net.minecraft.world.World;
+import net.minecraftforge.api.distmarker.Dist;
+import net.minecraftforge.api.distmarker.OnlyIn;
+import slimeknights.tconstruct.library.client.particle.AttackParticle;
+
+@OnlyIn(Dist.CLIENT)
+public class AxeAttackParticle extends AttackParticle {
+  public AxeAttackParticle(World world, double x, double y, double z, double xSpeed, double ySpeed, double zSpeed, IAnimatedSprite spriteList) {
+    super(world, x, y, z, xSpeed, ySpeed, zSpeed, spriteList);
+
+    this.particleScale = 0.4F;
+  }
+
+  @OnlyIn(Dist.CLIENT)
```

Comment:
Adding `@OnlyIn` to our own classes is not needed, as they will never be used outside of a server. Forge devs want us to keep usage of it to a minimal.